src/puppetlabs/ezbake/core.clj

```diff
@@ -699,6 +707,38 @@ Additional uberjar dependencies:
 (defmulti action
   (fn [action & args] action))
 
+(defn munge-deps
+  ([deps]
+   (into {} (concat (for [[k v] deps]
+                      (let [[project-name version] k
+                            dependencies (munge-deps v)
+                            metadata (if (empty? dependencies) {"version" version} {"version" version "dependencies" dependencies })
+                            ]
+                        {(str project-name) metadata}))))))
+
+(defn generate-build-metadata-files
+  [lein-project]
+  (let [dep-map (deputils/generate-dependency-map lein-project)
+        deps (munge-deps dep-map)
+        metadata {"packaging_type" {"ezbake" (get-ezbake-version lein-project)}
+                  "git_sha" (get-git-sha)
+                  "build_time" (new java.util.Date)
```

Comment:
I decided on rfc822, which should be parsable by machines but also human readable enough to get the relevant information across. I'm assuming that both future tooling and humans will be using this field.